src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,82 @@
 import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  todos: Todo[];
+  openTodo: (todo: Todo) => void;
```

Comment:
Is it open some todo? 
I think this function only provides todo to the selectedTodo state